server/plugin/taskLinkPreview.go

```diff
@@ -159,3 +159,44 @@ func (p *Plugin) PostBuildDetailsPreview(linkData []string, link, userID, channe
 	model.ParseSlackAttachment(post, []*model.SlackAttachment{attachment})
 	return post, ""
 }
+
+func (p *Plugin) PostReleaseDetailsPreview(linkData []string, link, userID, channelID string) (*model.Post, string) {
+	organization := linkData[3]
+	project := linkData[4]
+	releaseID := strings.Split(linkData[5], "&")[1][10:]
+	releaseDetails, _, err := p.Client.GetReleaseDetails(organization, project, releaseID, userID)
+	if err != nil {
```

Comment:
```suggestion
	if err != nil {
	p.API.LogDebug(.....
```
I think we should log the error with debug level.